test/jdk/sun/security/pkcs11/PKCS11Test.java

```diff
@@ -497,13 +521,17 @@ public static String getNssConfig() throws Exception {
         }
 
         String base = getBase();
+        String nss = "nss";
+        Path nssDirSource = Path.of(base).resolve(nss);
+        String nssDirDestination = Path.of(".").resolve(nss).toString();
```

Comment:
Using current directory here may be incorrect. Can this be changed to `Path.of(System.getProperty("test.classes"), ".")`

Reply:
The "test.classes" directory contains the compiled classes according to [jtreg doc](https://openjdk.org/jtreg/tag-spec.html#testvars).
Whereas, the current directory is the [Jtreg scratch](https://openjdk.org/jtreg/faq.html#scratch-directory) directory, which is the right place for the tests temporary files. Also, Jtreg will automatically clean this directory when the test begins.
The scratch directory does seem like the best place to store these temporary configuration files.